code/components/citizen-devtools/src/ResourceTimeWarnings.cpp

```diff
@@ -139,6 +201,27 @@ static InitFunction initFunction([]()
 			}
 			ImGui::End();
 			ImGui::PopFont();
+		}
+		else if ((usec() - lastHitch) < 5s)
+		{
+			ImGui::PushFont(GetConsoleFontTiny());
+
+			const float DISTANCE = 10.0f;
```

Comment:
A bit of code dup - maybe would be worthwhile to make a helper function for this kind of warning popup?

I'd have reused the resource warning string stuff but timing as well as phrasing seem ingrained there.